docs/digging-deeper/advanced-markdown.md

```diff
@@ -167,21 +167,14 @@ For all the supported syntax options, Hyde will resolve a relative link based on
 
 Since this happens in our Markdown processor, this will work regardless of if you have custom HTML or Blade enabled for your site.
 
-#### Resolving a route
-
 ```markdown
+<!-- Resolving a route -->
 [Home](hyde::route('home'))
-```
 
-#### Resolving a relative link
-
-```markdown
+<!-- Resolving a relative link -->
 [Home](hyde::relativeLink('home'))
-```
-
-#### Resolving a media asset
 
-```markdown
+<!-- Resolving a media asset -->
 ![Image](hyde::media('image.jpg'))
```

Comment:
Should be `asset`

Reply:
https://github.com/hydephp/develop/commit/b441848ed0553a10fa891733db163539c572e2e1